microsoft/testsuites/performance/common.py

```diff
@@ -42,8 +42,8 @@ def perf_disk(
     filename: str,
     core_count: int,
     disk_count: int,
-    disk_setup_type: DiskSetupType,
-    disk_type: DiskType,
+    disk_setup_type: Optional[DiskSetupType],
```

Comment:
1. Give the default value here, instead of set None when calling. `disk_setup_type: DiskSetupType = DiskSetupType.unknown,`
2. Add unknown for both enums, and use it.

Reply:
Done